lighthouse-core/config/config-plugin.js

```diff
@@ -164,6 +216,7 @@ class ConfigPlugin {
     assertNoExcessProperties(invalidRest, pluginName);
 
     return {
+      ...pluginGroupsJson && {groups: ConfigPlugin._parseGroups(pluginGroupsJson, pluginName)},
```

Comment:
is it possible to just do away with the `...pluginGroupsJson && {` construct here too?